content/inferno/soft-skills/impactful-presentations.md

```diff
@@ -0,0 +1,146 @@
+# Impactful Presentations
+
+## Why do you need to impress your audience?
+
+In my day, in the ancient Rome, data scientists were called [haruspexes](https://en.wikipedia.org/wiki/Haruspex). 
+The state-of-art technique they used to represent data and make decisions based on it was to spread the entrails of a bird on a sacred table and try to interpret them. 
+"Such a small stomach? Aha! I'm going to tell the general now that it's time to attack decisively!"
+
+But they always had that problem that they couldn't exceed an average success of 50 percent!
+Today, fortunately, we have more effective tools and the core field "Data Science" is looking at the factual evidence of the past to try to make better decisions in the present that affect the future. In the age of data-driven decisions, it is increasingly important to have a clear representation of them and how they advise us to act in practice. 
+
+Let us give a few examples:
+
+E-Commerce may want to understand which products are best sold, to what kind of target, in what volumes. 
+It may want to understand which products are often bought together so you can 
+get the famous "hey, the other users who bought X also bought Y!".\
+A newspaper might be interested in how age groups are divided and in what numbers, among its readers, 
+and might also want to see at what time of day its articles are consulted and from what type of device (mobile, desktop, real paper?).\
+A bank wants to understand what are the maximum margins that can be put on policies and loans, to find the best compromise between competitiveness and gain. These are just a few examples, now let's try to make a game:
+try to think of any kind of human activity, involving more than 5 humans, that would benefit from a clear representation
+of the data of their activity, to make actual decisions. 
+
+You will find that this reasoning applies to anything, _if you have enough data!_
+
+Extracting knowledge from the data, however, is useless if then the audience 
+to which we must communicate(managers, customers, colleagues, departments) 
+does not perceive the urgency of looking at the data to make decisions!
+
+With this guide, I want to introduce you to the key principles for effectively presenting your data and the conclusions
+that can be drawn from them. Remember! Every decision you propose will always have a certain risk of not being the right one 
+(the real world is based on unpredictable and chaotic mechanics and interactions), but the important point in all this is to 
+get as far away as possible from the success achieved by the haruspex and their weak and random 50 percent.
+
+During this guide, I consider that you're presenting with the tech support of simple slides.
+
+## Prerequisites
```

Comment:
If this guide is about time-series what is the purpose of this section?

By now we could have started to talk/explain concepts or ideas about time-series.

Why is this guide part of the Time-series PR?

Reply:
because I added it together, but it was a mistake in fact